### DISCUSSION TITLE
Generic Q on Nav radars

### DISCUSSION THREAD

**Discussion by Parel803** [PygmalionOfCyprus/cmo-db-requests#3891] [General]
If a class in the DB has a generic Navigations RADAR(s) and there is information available on more specific names/types. Is that worth while to add. Or is generic for NA RADARS sufficient?
best regards GJ

**Comment by PygmalionOfCyprus** [comment]
Yeah, I always try to be specific with nav radars when possible. Does it matter? No, not really. Is it cute and adds credibility to the sim? Yeah. So, if you have the info available, by all means.

**Comment by Parel803** [comment]
Thank you for your time and answer. I'll try to comply.
regards GJ
